### PR TITLE
D8CORE-635: Style Tweaks

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-635_

# Needed By (Date)
- Sprint close

# Urgency
- Styling fixes

# Steps to Test

- Checkout this branch
- Navigate to any page with an H1 
- Verify you see the larger style

- Create or find a quote through the WYSIWYG
- Verify it has larger font

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-635

## Folks to notify
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/scss/base/_type-a.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+// Style guide: Mixins.Typography.type-a
+//
+@mixin type-a {
+  @include modular-typography(5);
+  letter-spacing: -0.016em;
+
+  @include grid-media-max('sm') {
+    font-size: modular-scale(5) * $su-displays-mobile-factor;
+  }
```
Comment (cjwest) [inline src/scss/base/_type-a.scss]: @kerri-augenstein,
Do I change both the `modular-typography` and the `modular-scale`?

Comment (sherakama) [reply src/scss/base/_type-a.scss]: @cjwest 
The `modular-typography()` mixin is a wrapper for bourbon's `modular-scale()` where we get a font-size back. See: https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/mixins/modular-scale/_modular-typography.scss#L21

So what you are doing here looks good. 

I would make the suggestion to move this mixin override file to a utilities section that mirror's Decanters file paths.

**Comment by cjwest** [comment]
Hi @sherakama,
Thanks for pointing out where to place that file!
Please take a look and let me know if there is anything else.

**Comment by sherakama** [comment]
@cjwest Grab me on Friday to go over the changes in more detail.

**Pull request merged**